tests/test_utils.py

```diff
@@ -67,20 +66,3 @@ def test_scroll_slow_element_not_scrollable(mocker):
 
     # Ensure it detected non-scrollable element
     mock_driver.execute_script.assert_not_called()
-

```

Comment:
Suggest updating tests rather than removing.
These tests are for regression. Add testcases for newly added code / modules

Reply:
thanks, I have added two new test cases